web/cobrands/fixmystreet/base.scss

```diff
@@ -33,69 +33,102 @@
 }
 
 .confirmation-header {
-  padding: 2em 1em;
+  padding: 2em 0;
+  padding-top: 140px;
+  text-align: center;
+  background: transparent url(/cobrands/fixmystreet/images/tick-in-circle-100px.png) center 1em no-repeat;
```

Comment:
You could put 'default' as your top ALLOWED_COBRANDS and visit your local instance with 'default' in the URL and then it would show you what it looked like there.